Cargo.toml

```diff
@@ -6,3 +6,6 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+serde = { version = "*", features = ["derive"] }
+serde_yaml = "*"
```

Comment:
better to specify full version dependency. For serde at  the time of writing it is `serde = "1.0.203"`